egeaML/datareader.py

```diff
@@ -69,3 +69,23 @@ def plot_column_distribution(self, variable_name, title_plot, yticklabels):
             yticklabels=yticklabels
         )
         plt.title(title_plot)
+
+
+class FinancialDataReader:
+    #TODO: to be improved
+
+    def __init__(self, stock_name, start_date, end_date):
```

Comment:
Add optional parameter: **self.interval** 1m,2m,5m,15m,30m,60m,90m,1h,1d,5d,1wk,1mo,3mo

Reply:
Thanks @thomas-compagnoni for the comment.
I like the idea: we can discuss about it and make it real in another PR. This PR is to create a skeleton.